nupicaudio/encoders/wave.py

```diff
@@ -0,0 +1,28 @@
+# WAV encoder - use any available transformators
+
+from iteration import IterationEncoder
+
+from enum import Enum
+
+class WAVEncoder(IterationEncoder):
+
+  MODE = Enum('MODE', 'SCIPY') # TODO add others, matlab, ...
```

Comment:
someone worked with MIDI files here, can you add/show me the code for parsing a midi, so we could integrate it in this encoder? (later, in a new PR)